coss/base/templates/wagtailadmin/admin_base.html

```diff
@@ -0,0 +1,17 @@
+{% extends "wagtailadmin/admin_base.html" %}
+{% load staticfiles wagtailadmin_tags %}
+
+{% block css %}
+    <link rel="stylesheet" href="{% static 'wagtailadmin/css/vendor/jquery-ui/jquery-ui-1.10.3.verdant.css' %}">
+    <link rel="stylesheet" href="{% static 'wagtailadmin/css/vendor/jquery.tagit.css' %}">
+    <link rel="stylesheet" href="{% static 'wagtailadmin/css/core.css' %}" type="text/css">
+    {% hook_output 'insert_global_admin_css' %}
```

Comment:
Do we need to add these here? Can't we just add the custom css in the extra_css block?

Reply:
Moreover you can add css files through the `insert_global_admin_css` hook. This way we won't need to override and maintain the `admin_base` template.